cmd/podmanV2/containers/wait.go

```diff
@@ -42,7 +42,7 @@ func init() {
 	flags.DurationVarP(&waitFlags.Interval, "interval", "i", time.Duration(250), "Milliseconds to wait before polling for completion")
 	flags.BoolVarP(&waitFlags.Latest, "latest", "l", false, "Act on the latest container podman is aware of")
 	flags.StringVar(&waitCondition, "condition", "stopped", "Condition to wait on")
-	if registry.EngineOpts.EngineMode == entities.ABIMode {
+	if registry.EngineFacts.EngineMode == entities.ABIMode {
```

Comment:
I really dislike this name. What was wrong with EngineOpts?

Reply:
Since I was combining to structs, I came up with a 3rd name.  I can use EngineOpts.  No strong opinion.

Reply:
SGTM,